torchaudio/transforms.py

```diff
@@ -199,7 +199,7 @@ def __init__(self, sample_rate=16000, n_fft=400, win_length=None, hop_length=Non
                                        hop_length=self.hop_length,
                                        pad=self.pad, window_fn=window_fn, power=2,
                                        normalized=False, wkwargs=wkwargs)
-        self.mel_scale = MelScale(self.n_mels, self.sample_rate, self.f_min, self.f_max)
+        self.mel_scale = MelScale(self.n_mels, self.sample_rate, self.f_min, self.f_max, self.n_fft // 2 + 1)
```

Comment:
Is there a way to read off the size of the spectrogram instead of recomputing it?

Reply:
Not unless the spectrogram computes a specgram and we look at the tensor's dimension

Reply:
If that constant `n_fft // 2 + 1` is used in a few places within Spectrogram, we could attach it to Spectrogram. If that's our inferred default dimension, we could also just attach a `default_dimension` to Spectrogram, in case that ever changes, but that would add clutter to the Spectrogram interface.

Alright, I'm ok with having the value coded there. If there's ever a change in this, the test will fail :)